### PR TITLE
136 8 documentation + token generator testing code

### PR DESCRIPTION
**Pull request by frimic-solarly**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -34,3 +34,27 @@
...
+By default, the OSM is deactivated, which means it doesn't provide any energy on its output.
+If the initial default configuration has been applied, you can test tokens available on the Excel sheet "Token examples.xlsx" (Find it here: OpenSmartMeter\firmware\test).
+If you want to create other token:
+1) Open the repository on an IDE with python3.9 or above.
+2) Install the python requirements (in OpenSmartMeter/firmware/test):
+   pip3 install -r requirements.txt
+3) Change initial parameters in "test_openpaygo.py"
+Token_value can be days or kWh, depending on the initial default configuration you have defined on the OSM itself.
+4) Run script and get your desired token in console output.
+
```
Comment (dmohns) [inline docs/index.md:60]: Could you move these to a new section in the documentation? For example a quick-start guide, here is an example in our AirLink project 
image

Comment (frimic-solarly) [reply docs/index.md]: Done ;)

**Review by dmohns — COMMENTED** [review]
Few more comments:

- Can you rebase/merge latest `main` branch? I added some changes to the linter configuration
- Can you format your Python code using [Black](https://github.com/psf/black)?
- Would it be possible to have a CSV instad of XLSX file?

**Comment by frimic-solarly** [comment]
> Few more comments:
> 
>     * Can you rebase/merge latest `main` branch? I added some changes to the linter configuration
> 
>     * Can you format your Python code using [Black](https://github.com/psf/black)?
> 
>     * Would it be possible to have a CSV instad of XLSX file?

Hello @dmohns, 
I have rebased the branch and reformatted with Black. 
However for your third point, I think having a CSV instead of XLSX will not look good. Maybe just a .txt file or I move it to the documentation?

**Comment by dmohns** [comment]
Great thanks, can you now fix the remaining lint errors?

> I think having a CSV instead of XLSX will not look good. 

What do you mean by will not look good? Github automatically renders CSV file, for example: https://github.com/EnAccess/OpenPAYGO-Token/blob/main/documentation/examples/example_device_list.csv

File: docs/quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+By default, the OSM is deactivated, which means it doesn't provide any energy on its output.
+If the initial default configuration has been applied, you can test tokens available on the Excel sheet "Token examples.xlsx" (Find it here: OpenSmartMeter\firmware\test).
+If you want to create other token:
+
+1. Open the repository on an IDE with python3.9 or above.
+2. Install the python requirements (in OpenSmartMeter/firmware/test):
+   pip3 install -r requirements.txt
+3. Change initial parameters in "test_openpaygo.py"
+Token_value can be days or kWh, depending on the initial default configuration you have defined on the OSM itself.
+4. Run script and get your desired token in console output.
```
Comment (dmohns) [inline docs/quick-start-guide.md:24]: As discussed, maybe here you can just add the list of pre-generated tokens that user can use immediately to test the device.

The selected lines can probably go into `README.md` in the `firmware/test` directory (need to create it 😉 ), which explains how to run the test Python.

**Review by dmohns — APPROVED** [review]

**Pull request merged**